lib/open_weather/models/model.rb

```diff
@@ -10,7 +10,8 @@ class Model < Hashie::Trash
       attr_reader :options
 
       def initialize(args = nil, options = {})
-        super args
+        transformed_args = args.respond_to?(:transform_keys) ? args.transform_keys(&:to_s) : args
```

Comment:
Care to explain?

Reply:
A symbol-keyed hash was “ignored” by the model, I think because a Hashie::Dash property will account for whether it’s defined as a string or symbol (though there didn’t seem to be an issue retrieving the property via a method)

Reply:
it's possible that the model class needs a [merge initializer](https://github.com/hashie/hashie#mergeinitializer) instead. Either way make sure to have a spec for this and we can fix it later.

Reply:
Finding this hard to test on Model itself, would it be sufficient to test Station for now?

Reply:
Yes.